### PR TITLE
new test-client-pod for routing testing

### PR DESCRIPTION
**Pull request by lihongan**
The purpose of the PR is to add new test-client-pod that reusing the (nginx-alpine) image that owned by network edge team, and to decouple the hello-sdn image that used by pod-for-ping.
With the PR, we are expecting to reduce/minimize the risk of updating test image that not owned by ourselves. 
 
No new image is introduced, just copied and updated the step
```
Given /^I have a pod-for-ping in the#{OPT_QUOTED} project$/ do |project_name|
``` 

log:
https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/Runner/220045/console

@quarterpin @melvinjoseph86 @ShudiLi  Please help take a look. Thanks,

### CODE REVIEW

**Comment by lihongan** [comment]
I just updated one case to ensure the step is working, will ask team to help update all other related cases. Thanks.
cc @liangxia

File: features/step_definitions/helper_services.rb [hunk 1]
```diff
@@ -364,6 +364,25 @@
   end
 end
 
+# test-client-pod is a pod that has web client tools (curl, etc)
+# this pod is only used by Network Edge team for routing test
+Given /^I have a test-client-pod in the#{OPT_QUOTED} project$/ do |project_name|
```
Comment (liangxia) [inline features/step_definitions/helper_services.rb:369]: We can integrate the new step into `I have a pod-for-ping in the#{OPT_QUOTED} project` as most of the steps are the same

**Comment by lihongan** [comment]
logs: 
https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/Runner/222260/console
OCP-12562: using test-client-pod
OCP-12564: using pod-for-ping
to ensure both work well.
